lib/dry/view/part.rb

```diff
@@ -18,9 +19,7 @@ class Part
 
       attr_reader :_context
 
-      attr_reader :_renderer
-
-      def initialize(name:, value:, renderer:, context: nil)
+      def initialize(name:, value:, renderer: nil, context: nil)
```

Comment:
Do you reckon we could change this to `renderer: DefaultRenderer.new` here? This'd make it more obvious what the default value is, and allow us to remove that extra `#_renderer` below.

Reply:
Also, while we're on this, some thoughts on naming...

I wonder if `MissingRenderer` might be better here than `DefaultRenderer`, since it helps convey the idea that this is just an object to stand in place for an otherwise missing real renderer. What do you think?

Reply:
Maybe `NullRenderer` ?

Reply:
@AMHOL Heh, no worries :) Yeah, that was also my thinking in avoiding the "null" in the name. Thanks for taking a look at this though!

Reply:
Hey @timriley! Yeah I prefer `MissingRenderer`. `DefaultRenderer` sounds like it would work in place of no renderer as the default, when that's not what we want here. I too had the thought of calling it `NullRenderer` heh, but as Andy says, it doesn't adhere to the pattern exactly.